tools/universe/package_builder.py

```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
 
-_jre_url = "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz"
+_jre_url = "https://downloads.mesosphere.com/java/openjdk-8u212b03-hotspot-linux-x64.tar.gz"
```

Comment:
Wouldn't jre's at https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=hotspot be sufficient ?

Reply:
Good catch! Updated to fetch JRE here.